database/crdb.ksy

```diff
@@ -0,0 +1,187 @@
+meta:
+  id: crdb
```

Comment:
"crdb" doesn't seem to be very widely used term. [Linux kernel wiki](https://wireless.wiki.kernel.org/en/developers/Regulatory/wireless-regdb) names it "wireless-regdb", which is available in text and binary forms. "CRDB" would be abbreviation for "Central Regulatory (Domain Agent) DataBase", and it's not used widely. In fact, google search for that rather reveals [CRDB Bank on Wikipedia](https://en.wikipedia.org/wiki/CRDB_Bank), stuff in [Redis](https://docs.redislabs.com/latest/rs/administering/creating-databases/create-active-active/), [Cosmic-Ray DataBase](https://lpsc.in2p3.fr/crdb/), etc.

Suggestion: rename this to "wireless_regdb".

Reply:
Renamed.